README.md

```diff
@@ -1,23 +1,33 @@
 
-## Usage
+Effortlessly deploy your application to Heroku from CircleCI. Installs the Heroku CLI and provides commands to login, create an app, deploy, and more.
 
-Example use-cases are provided on the orb [registry page](https://circleci.com/orbs/registry/orb/circleci/heroku#usage-examples). Source for these examples can be found within the `src/examples` directory.
+---
 
 ## Resources
 
-[CircleCI Orb Registry Page](https://circleci.com/orbs/registry/orb/circleci/heroku) - The official registry page of this orb for all versions, executors, commands, and jobs described.  
-[CircleCI Orb Docs](https://circleci.com/docs/2.0/orb-intro/#section=configuration) - Docs for using and creating CircleCI Orbs.  
+[CircleCI Orb Registry Page](https://circleci.com/orbs/registry/orb/circleci/heroku) - The official registry page of this orb for all versions, executors, commands, and jobs described.
 
-### How To Contribute
+[CircleCI Orb Docs](https://circleci.com/docs/2.0/orb-intro/#section=configuration) - Docs for using, creating, and publishing CircleCI Orbs.
 
-We welcome [issues](https://github.com/CircleCI-Public/heroku-orb/issues) to and [pull requests](https://github.com/CircleCI-Public/heroku-orb/pulls) against this repository!
+### How to Contribute
 
-To publish a new production version:
-* Create a PR to the `Alpha` branch with your changes. This will act as a "staging" branch.
-* When ready to publish a new production version, create a PR from `Alpha` to `master`. The Git Subject should include `[semver:patch|minor|release|skip]` to indicate the type of release.
-* On merge, the release will be published to the orb registry automatically.
+We welcome [issues](https://github.com/CircleCI-Public/heroku-orb/issues) to and [pull requests](https://github.com/CircleCI-Public/heroku-orb/pulls) against this repository!
 
-For further questions/comments about this or other orbs, visit the Orb Category of [CircleCI Discuss](https://discuss.circleci.com/c/orbs).
+### How to Publish An Update
+
+1. Merge pull requests with desired changes to the main branch.
+   - For the best experience, squash-and-merge and use [Conventional Commit Messages](https://conventionalcommits.org/).
+2. Find the current version of the orb.
+   - You can run `circleci orb info circleci/heroku | grep "Latest"` to see the current version.
+3. Create a [new Release](https://github.com/CircleCI-Public/heroku-orb/releases/new) on GitHub.
```

Comment:
Would it make more sense to link to docs? I'd imagine this link would be invalid for end users

Reply:
Confirmed, redirects to sign in. 

Replace with this? https://docs.github.com/en/repositories/releasing-projects-on-github/managing-releases-in-a-repository

Reply:
This is from the template. But also, this would be for maintainers to create a new release, this brings them to the new release page for this project.

Reply:
Ah gotcha! Sounds good 🙌